relationships/candidates/REDIS.yml

```diff
@@ -8,9 +8,9 @@ lookups:
     tags:
       matchingMode: ALL
       predicates:
-        - tagKeys: ["aws.elasticache.endpointAddress", "aws.elasticache.configurationEndpointAddress"]
+        - tagKeys: ["aws.elasticache.endpointAddress", "aws.elasticache.configurationEndpointAddress", "aws.elasticache.primaryEndpointAddress", "aws.elasticache.readerEndpointAddress" ]
```

Comment:
Does `matchingMode: ALL` make sense in this context or would it need to be `matchingMode: ANY` ?
([docs](https://github.com/newrelic/entity-definitions/blob/main/docs/relationships/candidate_relationships.md#tags-mode))

Reply:
I guess, the matching mode refers to the predicates. the endpoint and port, it should match all.